config/rbac/role.yaml

```diff
@@ -6,19 +6,6 @@ metadata:
   creationTimestamp: null
   name: kserve-manager-role
 rules:
-- apiGroups:
```

Comment:
these role bindings should not be removed?

Reply:
@yuzisun These role are removed when I ran make `deploy-dev` so I added.

Reply:
oh maybe we should look at why `deploy-dev` removed those

Reply:
@yuzisun I added missing rbac kubebuilder steps in beta1 controller. Now the roles are not ignored.